floodviz/hydrograph_utils.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+import operator
+import time
+
+
+def parse_hydrodata(jdata, newd, idx, sites_value_maxes):
```

Comment:
Need to add comments describing the function, the inputs, and what it returns.Generally we use sphinx standards. You can go here for an example: https://pythonhosted.org/an_example_pypi_project/sphinx.html#function-definitions

It is preferable to write functions that don't have side effects - in other words they don't change their inputs or global data. For this function, I was expecting it to return the parsed data. Looking at it more closely it looks like it's parsing and returning a max val. Does it make sense to split these into two functions? If not consider returning a dictionary containing a key for the parsed data and a key for the max value.